Drivers/Inc/Display.h

```diff
@@ -37,10 +36,6 @@ typedef enum{
 	DISPLAY_ERR_OTHER     =-10  // Other nextion display error
 } DisplayError_t;
```

Comment:
Should this be incremental similar to the enum in UpdateDisplay.h or is this for visual purposes?

Reply:
Good point- I think it's negative because the display (which theoretically should send us error messages) uses negative values, but since the display isn't sending us messages anyways, maybe we can change this or just remove most of them altogether. @IshDeshpa what are your thoughts on deleting most of these and only leaving the errors we send ourselves (as default enum values)?

Reply:
Done!